### PR TITLE
change the gas token to be the native token of the network 

### PR DESCRIPTION
**Pull request by aguschirico**
### Title of pull request

The pull request should be described here

### Description

- Describe here your pull request modifications
- It could be more than one thing

### How to Test

- Config your `.env` file
- Run the server

#### Case 1

1. Run the server
```shell
$~ npm run serve
```

2. Click on NFT

__Expected Result__
- The nft green block should change

### Checklist
- [] Lint is clean `npm run lint`
- [] Prettier is passing `npm run prettier`

### CODE REVIEW

File: src/store.js [hunk 1]
```diff
@@ -99,6 +100,7 @@ export const store = {
   },
   async initMainSettings(chainId, rskConfig, sideConfig) {
     const state = store.state
+    state.gasToken = 'BTC' // TODO: In the future, the UI will support other tokens to pay for the gas. Currently this is only possible using rBTC.
```
Comment (pmprete) [inline src/store.js:103]: gasToken depends on the chainId, if you are in Ethereum is ETH. If it's on RSK is BTC if it's on Binance is BNB

File: src/components/crossForm/CrossForm.vue [hunk 2]
```diff
@@ -755,14 +755,13 @@ export default {
...
       const web3 = isRsk ? this.sharedState.sideWeb3 : this.sharedState.rskWeb3
-      const networkConf = isRsk ? this.sharedState.sideConfig : this.sharedState.rskConfig
       web3.eth.getGasPrice().then(gasPrice => {
         const costInWei = new BigNumber(ESTIMATED_GAS_AVG)
           .multipliedBy(gasPrice)
           .shiftedBy(-18)
           .toPrecision(6)
           .toString()
-        this.claimCost = `${costInWei} ${networkConf?.mainToken?.symbol}`
+        this.claimCost = `${costInWei} ${this.sharedState.rskConfig?.tokenPrefix}${this.sharedState.gasToken}`
```
Comment (pmprete) [inline src/components/crossForm/CrossForm.vue:764]: This should be the destination network config gasToken

**Review by pmprete — CHANGES_REQUESTED** [review]
The changes requested are because we don't want to change the gas token to always be rBTC, but when crossing from Ethereum to  RSK it was showing the gas price in ETH when it should show it on rBTC

File: src/store.js (hunk 1 above)
Comment (aguschirico) [reply src/store.js]: fixed.

File: src/components/crossForm/CrossForm.vue (hunk 2 above)
Comment (aguschirico) [reply src/components/crossForm/CrossForm.vue]: got it. Done.

File: src/constants/networks.js [hunk 3]
```diff
@@ -38,6 +41,7 @@ export const TEST_NET_BINANCE_CONFIG = {
   tokenPrefix: 'b',
   env: ENVIRONMENTS.TESTNET,
   mainToken: TEST_NET_BINANCE_MAIN_TOKEN,
+  gasToken: TEST_NET_BINANCE_MAIN_TOKEN,
```
Comment (pmprete) [inline src/constants/networks.js:44]: I think gasToken is  a different token than mainToken, this looks like a copy paste error
```suggestion
  gasToken: TEST_NET_BINANCE_GAS_TOKEN,
```

File: src/constants/networks.js [hunk 4]
```diff
@@ -59,6 +63,7 @@ export const TEST_NET_KOVAN_CONFIG = {
   tokenPrefix: 'e',
   env: ENVIRONMENTS.TESTNET,
   mainToken: TEST_NET_KOVAN_MAIN_TOKEN,
+  gasToken: TEST_NET_KOVAN_MAIN_TOKEN,
```
Comment (pmprete) [inline src/constants/networks.js:66]: same here
```suggestion
  gasToken: TEST_NET_KOVAN_GAS_TOKEN,
```

File: src/constants/networks.js [hunk 5]
```diff
@@ -79,9 +84,10 @@ export const TEST_NET_RINKEBY_CONFIG = {
   feePercentageDivider: 10_000,
   env: ENVIRONMENTS.TESTNET,
   mainToken: TEST_NET_KOVAN_MAIN_TOKEN,
+  gasToken: TEST_NET_KOVAN_MAIN_TOKEN,
```
Comment (pmprete) [inline src/constants/networks.js:87]: and here
```suggestion
  gasToken: TEST_NET_KOVAN_GAS_TOKEN,
```

File: src/constants/networks.js [hunk 6]
```diff
@@ -171,6 +180,7 @@ export const MAIN_NET_ETH_CONFIG = {
   tokenPrefix: 'e',
   env: ENVIRONMENTS.MAINNET,
   mainToken: MAIN_NET_ETHEREUM_MAIN_TOKEN,
+  gasToken: MAIN_NET_ETHEREUM_MAIN_TOKEN,
```
Comment (pmprete) [inline src/constants/networks.js:183]: and here
```suggestion
  gasToken: MAIN_NET_ETHEREUM_GAS_TOKEN,
```

File: src/constants/tokens/mainNetRskCrossEthereum.js [hunk 7]
```diff
@@ -1,9 +1,15 @@
...
+
+export const MAIN_NET_RSK_CROSS_ETHEREUM_GAS_TOKEN = {
   ...tokensInfo.TOKEN_RBTC_INFO,
-  symbol: 'RBTC',
+  symbol: 'rBTC',
+}
+
+export const MAIN_NET_RSK_CROSS_ETHEREUM_MAIN_TOKEN = {
+  ...tokensInfo.TOKEN_WETH_INFO,
+  symbol: 'rETH',
```
Comment (pmprete) [inline src/constants/tokens/mainNetRskCrossEthereum.js:12]: Here is just `symbol: 'ETH'`

File: src/constants/tokens/testNetRskCrossBinance.js [hunk 8]
```diff
@@ -1,6 +1,11 @@
 import * as tokensInfo from '../tokensInfo'
 import * as methodType from '../methodType'
 
+export const TEST_NET_RSK_CROSS_BINANCE_GAS_TOKEN = {
+  ...tokensInfo.TOKEN_RBTC_INFO,
+  symbol: 'rBTC',
```
Comment (pmprete) [inline src/constants/tokens/testNetRskCrossBinance.js:6]: When grossing form RSK Testnet to Binance, the gas on the binance network is BNB so it should be
`symbol: BNB`

**Review by pmprete — COMMENTED** [review]
I left some comments, there aren't that big but they must be changed

File: src/constants/networks.js (hunk 3 above)
Comment (aguschirico) [reply src/constants/networks.js]: done.

File: src/constants/networks.js (hunk 4 above)
Comment (aguschirico) [reply src/constants/networks.js]: done

File: src/constants/networks.js (hunk 5 above)
Comment (aguschirico) [reply src/constants/networks.js]: done

File: src/constants/networks.js (hunk 6 above)
Comment (aguschirico) [reply src/constants/networks.js]: done

File: src/constants/tokens/testNetRskCrossBinance.js (hunk 8 above)
Comment (aguschirico) [reply src/constants/tokens/testNetRskCrossBinance.js:6]: this is used when crossing from Binance to RSK, so it's rBTC.

File: src/constants/tokens/mainNetRskCrossEthereum.js (hunk 7 above)
Comment (aguschirico) [reply src/constants/tokens/mainNetRskCrossEthereum.js:12]: this is used when you cross ETH to RSK so it's rBTC right?

File: src/constants/tokens/testNetRskCrossBinance.js (hunk 8 above)
Comment (pmprete) [reply src/constants/tokens/testNetRskCrossBinance.js:6]: you are correct

**Review by pmprete — APPROVED** [review]
LGTM

**Pull request merged**